src/webview/css/css.js

```diff
@@ -157,20 +159,22 @@ hr {
   flex-direction: row;
 }
 .stream-text,
-.topic-text,
+.topic-content,
 .private-header {
   padding: 0 0.5em;
   line-height: 2;
   white-space: nowrap;
-  overflow: hidden;
-  text-overflow: ellipsis;
 }
-.topic-text {
+.topic-content {
   flex: 1;
   padding-left: 0.5em;
   background: #ccc;
   min-width: 30%;
 }
+.topic-text {
+  overflow: hidden;
```

Comment:
You definitely want some margin on the left. 
Check out how longer topics almost blend with the date div
Try `0.5em` or `1em`

Reply:
> some margin on the left.

I agree, it's definitely blending in a lot. Also I assume you mean margin on the right of the topic-text?